variables.tf

```diff
@@ -119,3 +119,19 @@ variable "ecs_platform_version" {
   # TODO: when LATEST points to 1.4.0, change default to LATEST
   default = "1.4.0"
 }
+
+variable "tags" {
+  type        = map(any)
+  description = "Optional; Map of key-value tags to apply to all applicable resources"
+  default     = {}
+}
+variable "tags_ecs_task_definition" {
+  type        = map(any)
+  description = "Optional; Map of key-value tags to apply to the ecs task definition"
+  default     = {}
+}
+variable "tags_security_group" {
+  type        = map(any)
+  description = "Optional; Map of key-value tags to apply to the security group"
+  default     = {}
+}
```

Comment:
A part of me feels like putting in a basic default tag is a good idea. Something like `ProvisionedBy = terraform-easy-fargate` or something. At least off the top of my head, it might add some value to the average user to know that a resource was built by a thing that they used.

Reply:
Not saying that we should do that but maybe that's worth a quick discussion

Reply:
Hmm, I'd rather leave that as an exercise for the user.  It also would feel kinda weird pushing a minor update that changed nearly every deployed resource without asking.